lib/oli_web/views/layout_view.ex

```diff
@@ -26,6 +26,7 @@ defmodule OliWeb.LayoutView do
   alias OliWeb.Breadcrumb.BreadcrumbTrailLive
   alias Oli.Delivery.Paywall.AccessSummary
   alias Oli.Delivery.Sections
+  alias OliWeb.Router.Helpers, as: Routes
```

Comment:
Was this necessary to add? Thought that we already has access to `Routes` in the layout